harbors/management/commands/add_helsinki_harbors_images.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+from django.conf import settings
+from django.core.files import File
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+
+from harbors.models import Harbor
+
+
+class Command(BaseCommand):
+
+    def handle(self, **options):
+        updated_harbors = 0
+        missing_images = []
+
+        for harbor in Harbor.objects.all():
+            with transaction.atomic():
```

Comment:
There seems to be only one db affecting place (`harbor.image_file.save(...)`), so not sure if this atomic is actually doing anything in practice.

Reply:
good point! will get rid of it